CHANGELOG.adoc

```diff
@@ -19,6 +19,7 @@
 - https://github.com/eclipse-sirius/sirius-components/issues/1746[#1746] [forms] Rich-text editor toolbar on forms representations are not grayed when read-only
 - https://github.com/eclipse-sirius/sirius-components/issues/1744[#1744] [forms] Checkboxes on forms representations are not grayed when read-only
 - https://github.com/eclipse-sirius/sirius-components/issues/1639[#1639] [core] Support schemaLocation and extendedMetaData for JSON resource
+- https://github.com/eclipse-sirius/sirius-components/issues/1775[#1775] [core] Increase representation/document/project label length from 50 to 1024 characters maximum
```

Comment:
Do these even make sense in the UI?
I agree that we should not impose some arbitrary small limit on the name's length, but I'm not sure the different places the representation name can appear in our UI can actually handle long names. At least the tabs for currently opened representation do not handle long names very well IMO.

Reply:
For the tabs, a PR has been proposed recently: https://github.com/eclipse-sirius/sirius-components/pull/1774

Reply:
Long names in the UI have been handled in the last version of the PR